### PR TITLE
Resolving error: You have requested a non-existent service "request"

### PR DESCRIPTION
**Pull request by cmmata**
Hello,

I'm using your bundle because it's Symfony3 compatible, but I found an error in the web interface. It was showing this:

>   You have requested a non-existent service "request". Did you mean one of these:    "monolog.logger.request", "request_stack", "router.request_context"?

I have called Symfony's request in the new way, like it's called in several documentation pages like [this one](http://symfony.com/doc/current/book/controller.html#managing-the-session), and now it works. Do you think it's ok with that, or do you prefer any other solution?

### CODE REVIEW

**Comment by Ener-Getick** [comment]
Indeed your solution is the best ☺ could you remove the attribute instead of commenting it please ?

**Comment by cmmata** [comment]
I forgot it, sorry! Removed

File: Controller/JobController.php [hunk 1]
```diff
@@ -46,8 +44,8 @@ public function overviewAction()
         }
 
         $pager = new Pagerfanta(new DoctrineORMAdapter($qb));
-        $pager->setCurrentPage(max(1, (integer) $this->request->query->get('page', 1)));
-        $pager->setMaxPerPage(max(5, min(50, (integer) $this->request->query->get('per_page', 20))));
+        $pager->setCurrentPage(max(1, (integer) $request->get('page', 1)));
+        $pager->setMaxPerPage(max(5, min(50, (integer) $request->get('per_page', 20))));
```
Comment (Ener-Getick) [inline Controller/JobController.php]: Why did you remove ``->query`` ?

Comment (cmmata) [inline Controller/JobController.php]: If I'm not wrong, $request->query->get('page', 1) and  $request->get('page', 1) are equivalent. I thought it was some Symfony version's improvement, but I can't find where it was explained.. I only find [official API method](http://api.symfony.com/master/Symfony/Component/HttpFoundation/Request.html#method_get) and some Stackoverflow answers [like this one](http://stackoverflow.com/questions/9784930/how-to-get-the-request-parameters-in-symfony2). 

Is there any difference? I can bring it back if you prefer.

Comment (Ener-Getick) [inline Controller/JobController.php]: ``$request->get()`` fetches the argument either from the request path or from its body whereas ``$request->query->get()`` which only fetches it from its path.
The first one should not be used unless this is really needed, so I'd prefer to revert this change.

**Comment by Ener-Getick** [comment]
Thank you @cmmata !

**Comment by cmmata** [comment]
Okay, thank you for your comment! I'll remember that.
